doc/release/1.16.0-notes.rst

```diff
@@ -280,6 +280,13 @@ and other path-like objects in addition to a file object. Furthermore, the
 ``np.load`` function now also supports path-like objects when
 using memory mapping (``mmap_mode`` keyword argument).
 
+``ndrange`` a multi-dimensional range-like object
+-------------------------------------------------
+``ndrange`` now supersedes the ``ndindex`` object for generating multi-index
+iterators. ``ndrange`` behaves much like the Python 3 ``range`` object and
+allows multi-dimensional slicing, iteration, reversal and efficient containment
+lookup. ``ndindex`` will continue to be maintained for backward compatibility.
```

Comment:
Another suggestion here: Switch the order of the first two sentences. That way you introduce ndrange before explaining its relation to ndindex.

Reply:
Thanks